cfme/v2v/infrastructure_mapping.py

```diff
@@ -0,0 +1,304 @@
+import attr
+from navmazing import NavigateToAttribute
+from navmazing import NavigateToSibling
+from widgetastic.utils import ParametrizedLocator
+from widgetastic.utils import WaitFillViewStrategy
+from widgetastic.widget import ParametrizedView
+from widgetastic.widget import View
+from widgetastic_patternfly import Button
+from widgetastic_patternfly import SelectorDropdown
+from widgetastic_patternfly import Text
+from widgetastic_patternfly import TextInput
+
+from cfme.base.login import BaseLoggedInPage
+from cfme.exceptions import ItemNotFound
+from cfme.modeling.base import BaseCollection
+from cfme.modeling.base import BaseEntity
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.appliance.implementations.ui import navigator
+from cfme.utils.update import Updateable
+from cfme.utils.version import Version
+from cfme.utils.version import VersionPicker
+from widgetastic_manageiq import InfraMappingList
+from widgetastic_manageiq import InfraMappingTreeView
+from widgetastic_manageiq import MultiSelectList
+from widgetastic_manageiq import V2VPaginatorPane
+
+
+class InfrastructureMappingView(BaseLoggedInPage):
+    title = Text("#explorer_title_text")
+
+    @property
+    def in_explorer(self):
```

Comment:
Consider calling this `in_mapping_explorer` to be more explicit when additional views are using this for is_displayed